tool/tsh/common/app.go

```diff
@@ -151,11 +152,24 @@ func printAppCommand(cf *CLIConf, tc *client.TeleportClient, app types.Applicati
 			return trace.BadParameter("app is Azure Cloud but Azure identity is missing")
 		}
 
-		var args []string
+		azureApp, err := newAzureApp(tc, cf, appInfo)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		resourceArgumentName := "-u"
```

Comment:
nit: use long form consistency with `--resource-id`:

```suggestion
		resourceArgumentName := "--username"
```